### PR TITLE
Silly teal gloves for realsies this time :3

### PR DESCRIPTION
**Pull request by Thinbug0**
## About the PR

Added teal gloves to the ClothesMate

(had an earlier pr thats now closed for this that trough unexperienced whoopsies doopsies got utterly comically messed up and i dont wanna keep bothering people so much to try and fix it, given how small the addition is overall. Sorry if its a bad practice, wont happen again <3)

## Why / Balance

it felt weird that we are missing them

## Media

tealGloves

## Requirements

- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- add: Teal gloves can now be found at the ClothesMate!

### CODE REVIEW

File: Resources/Prototypes/Entities/Clothing/Hands/colored.yml [hunk 1]
```diff
@@ -89,6 +89,35 @@
   - type: Fiber
     fiberColor: fibers-blue
 
+# Blue Gloves
```
Comment (0x6273) [inline Resources/Prototypes/Entities/Clothing/Hands/colored.yml:92]:
```suggestion
# Teal Gloves
```

**Comment by bruhmogus** [comment]
i really want to ask how in gods name TEAL GLOVES have the MAINTAINER DISCUSSION tag

**Comment by 0x6273** [comment]
> i really want to ask how in gods name TEAL GLOVES have the MAINTAINER DISCUSSION tag

Yeah, I can think of reasons why this would be denied, like clothesmate having too much stuff in it already or something. But I'm not sure why that discussion would have to happen in secret instead of on github.

**Comment by slarticodefast** [comment]
> i really want to ask how in gods name TEAL GLOVES have the MAINTAINER DISCUSSION tag

Anything that is not a bugfix or refactor needs discussion thread for 3 days according to policy in case someone wants to complain.
Not really much in there for this case and we need to adjust the policy a little so small stuff like this can be merged faster.
Let me leak some secret internal debating :)
grafik

> But I'm not sure why that discussion would have to happen in secret instead of on github.

I make sure to read everything commented here on GitHub and include it as feedback. I heard there were open maintainer discussions at some point before I joined, but it did not work out that well. We don't want anyone to get hatemail over things like combat balance.

**Comment by slarticodefast** [comment]
Thank you for your contribution!

**Pull request merged**